### PR TITLE
Fix bug cannot create foreign key

### PR DESCRIPTION
**Pull request by danghung-dev**
### Description
If the foreign key is lower case, it can be created. But if it has any character uppercase. I cannot create. 
I fix it by simple modify `this.escapeName(fk.foreignKey)`

image

### Checklist

- [] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by elv1s** [comment]
Thank you for the contribution! Please sign the CLA
https://cla.strongloop.com/agreements/strongloop/loopback-connector-postgresql

The change looks good, but I think you need to update the tests as well
https://github.com/strongloop/loopback-connector-postgresql/blob/master/test/postgresql.autoupdate.test.js#L630

**Comment by danghung-dev** [comment]
Thank you @elv1s 
I have already updated the tests and signed the CLA.

**Comment by elv1s** [comment]
Thanks @danghung-dev, it looks like the cla check is still failing. Did you sign the loopback-connector-postgres cla? It is different from the loopback cla

**Comment by danghung-dev** [comment]
image
Please check screenshot. 
Maybe I did something wrong?

**Comment by dhmlau** [comment]
@slnode test please

**Comment by dhmlau** [comment]
@rmg, could you please help?  Looks like @danghung-dev has signed the CLA but it's not showing the right status? Thanks!

**Comment by rmg** [comment]
This pull request includes a commit attributed to @smartlogvn, so they will also need to sign the CLA.

**Comment by danghung-dev** [comment]
Thanks @rmg 
Sorry, this is my mistake. I made commit but didn't check author.
I removed smartlogvn commit and force pushed again.

**Comment by elv1s** [comment]
@slnode test please

**Comment by dhmlau** [comment]
There is one test case failing:
```
1) autoupdate
13:41:50        foreign key constraint
13:41:50          should create, update, and delete foreign keys:
13:41:50 
13:41:50       Uncaught AssertionError [ERR_ASSERTION]: error: duplicate key value violates unique constraint "pg_namespace_nspname_index"
13:41:50       + expected - actual
13:41:50 
13:41:50       -false
13:41:50       +true
13:41:50       
13:41:50       at /home/jenkins/workspace/nb/loopback-connector-postgresql~master/3ed5ebba/test/postgresql.autoupdate.test.js:632:9
13:41:50       at /home/jenkins/workspace/nb/loopback-connector-postgresql~master/3ed5ebba/node_modules/loopback-connector/node_modules/async/dist/async.js:473:16
13:41:50       at iteratorCallback (node_modules/loopback-connector/node_modules/async/dist/async.js:1062:13)
13:41:50       at /home/jenkins/workspace/nb/loopback-connector-postgresql~master/3ed5ebba/node_modules/loopback-connector/node_modules/async/dist/async.js:969:16
13:41:50       at /home/jenkins/workspace/nb/loopback-connector-postgresql~master/3ed5ebba/lib/migration.js:321:24
13:41:50       at cbForWork (node_modules/loopback-datasource-juggler/lib/observer.js:234:34)
13:41:50       at /home/jenkins/workspace/nb/loopback-connector-postgresql~master/3ed5ebba/node_modules/loopback-connector/lib/sql.js:643:9
13:41:50       at Query.callback (lib/postgresql.js:152:7)
13:41:50       at Query.handleError (node_modules/pg/lib/query.js:142:17)
13:41:50       at Connection.connectedErrorMessageHandler (node_modules/pg/lib/client.js:183:17)
13:41:50       at Socket.<anonymous> (node_modules/pg/lib/connection.js:125:12)
13:41:50       at addChunk (_stream_readable.js:263:12)
13:41:50       at readableAddChunk (_stream_readable.js:250:11)
13:41:50       at Socket.Readable.push (_stream_readable.js:208:10)
13:41:50       at TCP.onread (net.js:601:20)
13:41:50 
```

Since it's only failing for Node.js 8, let me kick off CI again to make sure.

**Comment by dhmlau** [comment]
@slnode test please

File: test/postgresql.autoupdate.test.js [hunk 1]
```diff
@@ -477,6 +477,9 @@ describe('autoupdate', function() {
           'customerId': {
             'type': 'String',
             'length': 20,
+            'postgresql': {
```
Comment (jannyHou) [inline test/postgresql.autoupdate.test.js]: If the columnName is the same as property name, why do we have to specify it again here?

Comment (danghung-dev) [reply test/postgresql.autoupdate.test.js]: We have to do that, if we don't the column name will be convert to lowercase. 
Please take a look at this issue. https://github.com/strongloop/loopback-connector-postgresql/issues/38

Comment (jannyHou) [reply test/postgresql.autoupdate.test.js]: I see thanks for pointing to the issue.

**Pull request merged**